sql/src/test/java/io/crate/integrationtests/PostgresJobsLogsITest.java

```diff
@@ -41,6 +42,7 @@
 
 @ESIntegTestCase.ClusterScope(numDataNodes = 1, numClientNodes = 0, supportsDedicatedMasters = false)
 @UseJdbc(1)
+@UseSemiJoins(0) // Avoid set session stmt to interfere with tests
```

Comment:
Here it is 0 but it doesn't execute with semi joins?

Reply:
This class purpose is to test the sys.jobs/sys.jobs_log table and the `set session semi_joins` statement interferes with the tests as it's logged in Jobs/JobsLog.